src/sp_cookie_encryption.c

```diff
@@ -42,13 +43,14 @@ static zend_string *encrypt_data(zend_string *data) {
 }
 
 PHP_FUNCTION(sp_setcookie) {
-  zval params[7] = {{{0}}};
   zend_string *name = NULL, *value = NULL, *path = NULL, *domain = NULL,
-              *samesite = NULL;
+#if PHP_VERSION_ID >= 70300
+              *samesite = NULL,
+#endif
+              *value2 = NULL, *path2 = NULL;
```

Comment:
Maybe a better naming scheme, like `value_enc` ?
`path2` could only be used if `PHP_VERSION_ID` < 70300, and maybe rename it to `path_samesite` or something like that?

Reply:
done